qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -0,0 +1,276 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Quantum Pegasos Support Vector Classifier."""
+
+import logging
+from datetime import datetime
+from typing import Optional, Dict, Tuple
+
+import numpy as np
+from sklearn.svm import SVC
+
+from qiskit_machine_learning.exceptions import QiskitMachineLearningError
+from qiskit_machine_learning.kernels.quantum_kernel import QuantumKernel
+
+logger = logging.getLogger(__name__)
+
+
+class PegasosQSVC(SVC):
+    """
+    This class implements Quantum Pegasos Support Vector Classifier algorithm developed in [1]
+    and includes overridden methods ``fit`` and ``predict`` from the ``SVC`` super-class.
+
+    **Example**
+
+    .. code-block::
+        quantum_kernel = QuantumKernel()
+
+        pegasos_qsvc = PegasosQSVC(quantum_kernel=quantum_kernel)
+        pegasos_qsvc.fit(sample_train, label_train)
+        pegasos_qsvc.predict(sample_test)
+
+    **References**
+        [1]: Shalev-Shwartz et al., Pegasos: Primal Estimated sub-GrAdient SOlver for SVM.
+            `Pegasos for SVM <https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf>`_
+
+    """
+
+    def __init__(
+        self,
+        quantum_kernel: Optional[QuantumKernel] = None,
+        C: float = 1,
+        num_steps: int = 1000,
+        precomputed: bool = False,
+    ) -> None:
+        """
+        Args:
+            quantum_kernel: QuantumKernel to be used for classification.
```

Comment:
I didn't include it on purpose since there's also nothing like this in the documentation of the regular QSVC and the behavior of None depends on the default behavior of the `QuantumKernel` class